Packages/com.unity.inputsystem/InputSystem/Plugins/XR/Devices/WindowsMR.cs

```diff
@@ -13,7 +13,7 @@ namespace UnityEngine.XR.WindowsMR.Input
     public class WMRHMD : XRHMD
     {
         [Preserve]
-        [InputControl(noisy = true)]
+        [InputControl]
```

Comment:
Unrelated fix for which I didn't bother opening a PR (esp. since this XR stuff has no associated tests). `userPresence` should not be classified as noisy. Was erroneously added by me.